lib/ansible/modules/network/netscaler/netscaler_servicegroup.py

```diff
@@ -271,6 +271,12 @@
                     - Server port number.
                     - Range C(1) - C(65535)
                     - "* in CLI is represented as 65535 in NITRO API"
+            state:
```

Comment:
I would prefer another name for this param `state` for consistency with other modules, which uses state for `present` or `absent`. I would suggest to use `enabled=<bool>`. Any other opinions?

Reply:
I just realized it is a suboption. LGTM